src/_variables.js

```diff
@@ -0,0 +1,7 @@
+export const breakpoints = {
+  xs: "480px",
+  sm: "768px",
+  md: "992px",
+  lg: "1200px",
+  xl: "1400px"
```

Comment:
Do we really care if screen is `1200px` or `1400px` ? Doesn't seem like a big difference for me.